### PR TITLE
fix(storage): remove targetIdentityId from upload  and copy api

### PR DESCRIPTION
**Pull request by kvramyasri7**
#### Description of changes

This change involves removing the targetIdentityId from `upload`, `copy` api

#### Checklist

- [X] PR description included
- [X] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by cshfang — DISMISSED** [review]

**Review by ashwinkumar6 — DISMISSED** [review]
LGTM
thank you

File: packages/storage/src/types/options.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export type StorageListPaginateOptions = StorageOptions & {
 	nextToken?: string;
 };
 
-export type StorageRemoveOptions = StorageOptions;
+export type StorageRemoveOptions = Omit<StorageOptions, 'targetIdentityId'>;
```
Comment (HuiSF) [inline packages/storage/src/types/options.ts:28]: One nit picking:
`StorageOptions` is still the category level type, should it include "identity id" this provider specific property?

**Review by HuiSF — APPROVED** [review]

File: packages/storage/src/types/options.ts (hunk 1 above)
Comment (AllanZhengYP) [reply packages/storage/src/types/options.ts:28]: @HuiSF We will standardize the interfaces in follow up PR to remove the `targetIdentityId` in upload/copy destination/remove API for both provider and category. 

CC @ashwinkumar6

**Review by AllanZhengYP — APPROVED** [review]
At this point,

**Review by HuiSF — APPROVED** [review]

**Pull request merged**